_posts/2018-01-05-Building-RetroArch.md

```diff
@@ -21,52 +21,21 @@ Credit to Mission20000 for the original work on these ports.
 a new copy of the emulator every time you want to add or remove roms.
 - Input / controls are still under development. They may work well, or not at all. 
 This is NOT a bug.
-- Switch Joycons must be used in detached mode (NOT connected to the sides of the
-Switch console otherwise the input / controls won't work.
-- You will probably need to reboot the console to quit the game / emulator.
+- Switch Joycons can be used in either detached or attached mode.
+- You will need to press both + and - at the same time to return to the RetroArch menu from inside a game
 - You can attempt to build other cores for RetroArch than Snes9x, but many will fail due to 
 requirements such as needing to use the file system, or needing specific graphics
 systems. Some cores for newer systems such as N64, PSX etc are not able to run yet, even 
-if you could build them.
+if you could build them. You can view a list of the cores and their status here: https://docs.google.com/spreadsheets/d/11Q21oCxj_xZ08QzpvwBuHDiuKkGjTFKRWRl2ZweFPCU/
 
 ## Snes9x-2010
 
-Before being able to build RetroArch, we need to build the Snes9x-2010 core. But before
```

Comment:
This could use a link to https://reswitchedweekly.github.io/Development-Setup/#libtransistor